src/server/bg_services/namespace_monitor.js

```diff
@@ -206,20 +208,41 @@ class NamespaceMonitor {
     }
 
     async test_nsfs_resource(nsr) {
+        dbg.log1('test_nsfs_resource: (name, namespace_store, nsfs_config):',
+            nsr.name, nsr.namespace_store, nsr.nsfs_config);
+        const missing_root_path_err = 'MissingRootPathError';
+        let entries;
+        let stat;
+        let error_details;
         try {
             const fs_context = {
                 backend: nsr.nsfs_config.fs_backend || '',
                 warn_threshold_ms: config.NSFS_WARN_THRESHOLD_MS,
             };
-            await nb_native().fs.readdir(fs_context, nsr.nsfs_config.fs_root_path);
-            const stat = await nb_native().fs.stat(fs_context, nsr.nsfs_config.fs_root_path);
+            entries = await nb_native().fs.readdir(fs_context, nsr.nsfs_config.fs_root_path);
```

Comment:
@liranmauda  Do you remember why we need here readdir and stat?
cc: @shirady

Reply:
I remember vagly, delete is not returning something that we can relay on.

Reply:
What we will lose if we leave only the `stat`?

Reply:
Guy is saying it's because readdir checks read permissions and in the past stat didn't check permissions (although it's checking read permissions today so now it's redundant)
but let's keep it as it is
@liranmauda thanks!